scripts/check-coverage.jsh

```diff
@@ -57,7 +57,7 @@ var moduleCoverageData = List.of(
   new ModuleCoverage("sirius-components-collaborative-formdescriptioneditors", 84.0),
   new ModuleCoverage("sirius-components-formdescriptioneditors-graphql", 59.0),
   new ModuleCoverage("sirius-components-trees", 83.0),
-  new ModuleCoverage("sirius-components-collaborative-trees", 65.0),
+  new ModuleCoverage("sirius-components-collaborative-trees", 64.0),
```

Comment:
The goal of this file is to never lower the expected code coverage value. This is why after the build, the continuous integration process must have indicated that additional tests were required.

I'll add at least one test to validate the behavior of this contribution but keep in mind that we will not accept contributions that lower our code coverage expectations.